planner/core/explain.go

```diff
@@ -857,6 +812,21 @@ func (p *DataSource) ExplainInfo() string {
 	return buffer.String()
 }
 
+// ExplainInfo implements Plan interface.
+func (p *PhysicalExchangeSender) ExplainInfo() string {
+	buffer := bytes.NewBufferString("ExchangeType: ")
+	switch p.ExchangeType {
+	case tipb.ExchangeType_PassThrough:
```

Comment:
why named `passThrough`?
it may menas collecting all data from server nodes to one node? the reverse of broadcase?

Reply:
pass through means transfer data from one node to another node,